src/main/resources/swagger/api-docs.yaml

```diff
@@ -434,13 +569,151 @@ paths:
   /api/methods:
     get:
       tags:
-      - Method Repository
+        - Method Repository
       operationId: listMethodRepositoryMethods
       summary: |
         Lists Method Repository methods.
+      # GAWB-1303
+      # parameters:
+      #   - name: namespace
+      #     in: query
+      #     description: Namespace of method.
+      #     required: false
+      #     type: string
+      #   - name: name
+      #     in: query
+      #     description: Name of method.
+      #     required: false
+      #     type: string
+      #   - name: snapshotId
+      #     in: query
+      #     description: Snapshot ID of method.
+      #     required: false
+      #     type: number
+      #   - name: synopsis
+      #     in: query
+      #     description: Synopsis of method.
+      #     required: false
+      #     type: string
+      #   - name: documentation
+      #     in: query
+      #     description: Documentation of method.
+      #     required: false
+      #     type: string
+      #   - name: owner
+      #     in: query
+      #     description: Owner of method.
+      #     required: false
+      #     type: string
+      #   - name: payload
+      #     in: query
+      #     description: Payload of method in WDL.
+      #     required: false
+      #     type: string
+      #   - name: entityType
+      #     in: query
+      #     description: Type of Method - Task or Workflow
+      #     required: false
+      #     type: string
+      #   - name: excludedFields
+      #     in: query
+      #     description: Fields to exclude in JSON response.
+      #     required: false
+      #     type: string
+      #   - name: includedFields
+      #     in: query
+      #     description: Fields to include in JSON response
+      #     required: false
+      #     type: string
+      responses:
+        200:
+          description: An array of methods.
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/MethodResponse'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: Add a Method.
+      description: |
+        Inserts a method into the method repository and
+        gives the owner full access permissions. Must supply a namespace
+        name, payload, and entityType. Cannot supply a snapshotId
+      tags:
+        - Method Repository
+      consumes:
```

Comment:
do we need a consumes attribute? the only other ones that have it are for multipart form data